backend/app/api/v1/endpoints/__init__.py

```
@@ -0,0 +1 @@
+from .auth import *
```

Ruff (0.8.2)

`from .auth import *` used; unable to detect undefined names

(F403)